firebase-config/src/main/java/com/google/firebase/remoteconfig/FirebaseRemoteConfig.java

```diff
@@ -66,10 +66,10 @@
    * <p>{@link FirebaseRemoteConfig} uses the default {@link FirebaseApp}, so if no {@link
    * FirebaseApp} has been initialized yet, this method throws an {@link IllegalStateException}.
    *
-   * <p>To identify the current app instance, the fetch request creates a Firebase Instance ID
-   * token, which periodically sends data to the Firebase backend. To stop the periodic sync, call
-   * {@link com.google.firebase.iid.FirebaseInstanceId#deleteInstanceId}. To create a new token and
-   * resume the periodic sync, call {@code fetchConfig} again.
+   * <p>To identify the current app instance, the fetch request creates a Firebase installation ID,
+   * which periodically sends data to the Firebase backend. To stop the periodic sync, call {@link
+   * com.google.firebase.installations.FirebaseInstallations#delete}. To create a new token and
```

Comment:
What means "token" in this context?
Is this a (FIS) authentication token?
If yes, do developers need to know about this implementation detail.
If no, can you please specify what "token" is meant here?

How about this:
"Note: If you send another Fetch request after the deletion, Remote Config and Firebase installations will create a new installation ID for this Firebase installation and resume the periodic sync."

Reply:
It's a FIS auth token, yes. The note might be helpful to give context to the `#delete` method developers can call, and give additional background for developers using a FIS auth token to identify test devices in the ABT console (where we call out that they need to use a FIS auth token). I think the note you wrote sounds good and provides the right level of context.